src/components/addComments.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <section class="add_message_section">
+    <form @submit.prevent="addMessageHandler" class="message_form">
+      <textarea
+        class="message_textarea"
+        v-model="currentTextAreaValue"
+        name="message"
+        id="message"
+        cols="30"
+        rows="10"
+        placeholder="Add a comment…"
+        required
+      />
+      <img
+        class="avatar"
+        width="32"
+        height="32"
+        :src="require(`../photos/photosForProfile/${currentUser.pathToPhoto}`)"
+        alt="avatar"
+      />
+      <button class="send_message" type="submit">SEND</button>
+    </form>
+
+    <select
+      @change="
+        (event) => {
+          changeCurrentUser(Number(event.target.value));
+        }
+      "
+      name="user_names"
+      id="user_names"
+    >
+      <option
+        v-for="(account, index) in allAccountsList"
+        :key="account.id"
+        :value="index"
+      >
+        {{ account.accountName }}
+      </option>
+    </select>
+  </section>
+</template>
+<script>
+import { v4 } from "uuid";
+import moment from "moment/moment";
+import { mapState, mapMutations } from "vuex";
+export default {
+  name: "AddComments",
+  data() {
+    return {
+      currentTextAreaValue: "",
+    };
+  },
+  methods: {
+    updateInput(event) {
+      this.currentTextAreaValue = event.target.value;
+    },
+    addMessageHandler() {
+      const newComment = {
+        id: v4(),
+        userId: this.currentUser.id,
+        text: this.currentTextAreaValue,
+        date: moment().format("YYYY-MM-DD HH:mm"),
```

Comment:
it's better to save timestamps because it's easier to parse them later. so save a timestamp here and then format it as you need when you use it
```suggestion
        date: Date.now(),
```